components/Cards/docs/theming-extensions.md

```diff
@@ -0,0 +1,37 @@
+### Theming Extensions
```

Comment:
Please rename to “Theming” and theming.md

Reply:
Addressed in a20a897 and 0503c1c. Description updated.